### PR TITLE
enable extensions for notebook 5.3 via conf.d files

### PR DESCRIPTION
**Pull request by minrk**
removes pre/post-link scripts

manual enabling may still be run for notebook versions earlier than 5.3

### CODE REVIEW

**Comment by minrk** [comment]
🤞 that I got the windows .bat right on the first go

File: recipe/bld.bat [hunk 1]
```diff
@@ -1,8 +1,8 @@
 REM enable extensions via conf.d files for notebook >= 5.3
-set ETC="%CONDA_PREFIX%\etc\jupyter"
-set SERVER_CONFIG="%ETC%\jupyter_notebook_config.d"
-set TREE_CONFIG="%ETC%\nbconfig\tree.d"
-if not exists "%SERVER_CONFIG%" mkdir "%SERVER_CONFIG%"
-if not exists "%TREE_CONFIG%" mkdir "%TREE_CONFIG%"
-copy "%RECIPE_DIR%\serverextension.json" "%SERVER_CONFIG%/ipyparallel.json"
-copy "%RECIPE_DIR%\nbextension.json" "%TREE_CONFIG%/ipyparallel.json"
+set ETC=%CONDA_PREFIX%\etc\jupyter
```
Comment (isuruf) [inline recipe/bld.bat]: `set "ETC=%CONDA_PREFIX%\etc\jupyter"` is what you want

Comment (minrk) [reply recipe/bld.bat]: Thank you!

File: recipe/meta.yaml [hunk 2]
```diff
@@ -23,6 +23,20 @@ requirements:
...
     - pip
+    - futures               # [py2k]
+    - decorator
+    - tornado >=4
+    - python-dateutil >=2.1
+    - pyzmq >=13
+    - ipython_genutils
+    - ipython >=4
+    - jupyter_client
+    - ipykernel
```
Comment (jakirkham) [inline recipe/meta.yaml]: Why are all of these needed?

Comment (minrk) [reply recipe/meta.yaml]: runtime dependencies are needed for the  `nbextension install` command.

**Pull request merged**